src/test/build.test.ts

```diff
@@ -191,4 +199,44 @@ test('finbot > starts a build > handles non-200 response', (t) => {
 
   // assert
   sinon.assert.calledWith(replyStub, `Could not start build. Got status code 403`);
+});
+
+test('finbot > start build > when no token set', (t) => {
+  // arrange

```

Comment:
![](http://img2.rnkr-static.com/list_img_v2/19540/339540/C520/the-very-best-of-the-futurama-fry-meme-u1.jpg)